apps/modernization-ui/src/apps/patient/add/PatientAddRouting.tsx

```diff
@@ -5,7 +5,7 @@ import { Outlet } from 'react-router-dom';
 import { BasicExtendedTransitionProvider } from 'apps/patient/add/useBasicExtendedTransition';
 import { AddPatientBasic } from './basic/AddPatientBasic';
 
-const PatientDataProviderWrapper = () => (
+export const PatientDataProviderWrapper = () => (
```

Comment:
does this need to be exported?

Reply:
good catch, made some changes and didn't need it anymore